src/components/field/select-field.jsx

```
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import { Label } from '@/components/ui/label';
+import { cn } from '@/lib/utils';
+import { capitalizeStr } from '@/utils/string';
+
+export function SelectField({ name, label, required = false, type, options, ...props }) {
+  const { errors, setFieldValue } = props;
```

ESLint

[error] 'errors' is missing in props validation

(react/prop-types)